src/python/stup/Config.py

```diff
@@ -7,7 +7,7 @@
 FIN_ACK_SEND_TIMES = 3
 
 BANDWIDTH = 3 * 1024 * 1024 # bytes per sec
-CHUNK_SIZE = 1000 
```

Comment:
the default TCP MSS is about 1000

May I know the reason for you to make that change?